rnd/autogpt_server/autogpt_server/blocks/ai.py

```diff
@@ -37,55 +31,56 @@ def __init__(self):
             input_schema=LlmCallBlock.Input,
             output_schema=LlmCallBlock.Output,
             test_input={
-                "config": {
-                    "model": "gpt-4-turbo",
-                    "api_key": "fake-api",
-                },
+                "model": "gpt-4-turbo",
+                "api_key": "fake-api",
                 "expected_format": {
                     "key1": "value1",
                     "key2": "value2",
                 },
-                "sys_prompt": "System prompt",
```

Comment:
Don't we need to keep sys_prompt in the test_input?

Reply:
Intentionally removed this to verify it still executes the block without the sys_prompt.